### PR TITLE
My updates to make Biff12Writer

### PR DESCRIPTION
**Pull request by mikel3361**
My updates to make multiple sheets, (only at the and of prevoius sheet.

### CODE REVIEW

File: src/cBiff12Writer.cls [hunk 1]
```diff
@@ -105,10 +109,11 @@ End Property
 ' Methods
 '=========================================================================
 
-Public Function Init( _
+Public Function init( _
```
Comment (wqweto) [inline src/cBiff12Writer.cls]: Can we keep `Init` function name casing?

File: src/cBiff12Writer.cls [hunk 2]
```diff
@@ -235,7 +256,7 @@ EH:
     Resume Next
 End Function
 
-Public Function AddImage(ByVal lColIndex As Long, baData() As Byte, ByVal lLeft As Long, ByVal lTop As Long, ByVal lWidth As Long, ByVal lHeight As Long)
+Public Function AddImage(ByVal lColIndex As Long, baData() As Byte, ByVal lLeft As Long, ByVal lTOp As Long, ByVal lWidth As Long, ByVal lHeight As Long)
```
Comment (wqweto) [inline src/cBiff12Writer.cls]: Can we keep `lTop` param name here too?

File: src/cBiff12Writer.cls [hunk 3]
```diff
@@ -248,19 +269,19 @@ Public Function AddImage(ByVal lColIndex As Long, baData() As Byte, ByVal lLeft
     baTemp = baData
     oImagePart.Contents = baTemp
     Set oDoc = oDrawingPart.XmlDocument
-    With oDoc.documentElement.appendChild(oDoc.createElement("xdr:twoCellAnchor"))
+    With oDoc.DocumentElement.appendChild(oDoc.createElement("xdr:twoCellAnchor"))
```
Comment (wqweto) [inline src/cBiff12Writer.cls:272]: `DocumentName` is better indeed

File: src/cBiff12Writer.cls [hunk 4]
```diff
@@ -277,7 +298,7 @@ Public Function AddImage(ByVal lColIndex As Long, baData() As Byte, ByVal lLeft
             With .appendChild(oDoc.createElement("xdr:blipFill"))
                 With .appendChild(oDoc.createElement("a:blip"))
                     Set oAttr = oDoc.createNode(2, "r:embed", STR_NS3)
-                    oAttr.nodeValue = m_oContainer.GetRelID(oDrawingPart, oImagePart)
+                    oAttr.NodeValue = m_oContainer.GetRelID(oDrawingPart, oImagePart)
```
Comment (wqweto) [inline src/cBiff12Writer.cls:301]: `NodaValue` is better too.

File: src/cBiff12Writer.cls [hunk 5]
```diff
@@ -127,7 +133,22 @@ Public Function Init( _
     ReDim m_uRowHdr.m_rgBrtColspan(0 To 0) As UcsBiff12BrtColSpanType
     m_uRowHdr.m_rgBrtColspan(0).m_colLast = m_lColCount - 1
     '--- success
-    Init = True
+    init = True
+End Function
+
+Public Function AddSheet(Optional ByVal SheetName As String = "", Optional ByVal lColCount As Long = 1) As Boolean
```
Comment (wqweto) [inline src/cBiff12Writer.cls]: `As String = ""` is redundant. Can you remove `= ""` please.

File: src/cBiff12Writer.cls (hunk 1 above)
Comment (mikel3361) [reply src/cBiff12Writer.cls]: Yes, Case sesitivity is not made by myself. I think vb6 IDE make the changes, maybe the german version i use.

File: src/cBiff12Writer.cls (hunk 2 above)
Comment (mikel3361) [reply src/cBiff12Writer.cls]: Changes not made by myself... Leave it unchanged

Comment (mikel3361) [reply src/cBiff12Writer.cls]: Take the real changes in init, addshhet and in biff12part and leave Case sesitivity by yourself. Vb does no Checks for that

File: src/cBiff12Writer.cls (hunk 5 above)
Comment (mikel3361) [reply src/cBiff12Writer.cls]: Optional parameters will better with a default value, but sorry i cant change it now. Its  weekend and the first days at work we  have to do  end  of month jobs

**Comment by mikel3361** [comment]
I made.changes in my fork directly in AddSheet using no default value. Using IsMissing for a check inside,  hope it works while cant test it yet

**Comment by mikel3361** [comment]
Files chabged

**Comment by mikel3361** [comment]
I don't see my changes. I will do pull it again

**Pull request closed without merging**